how-to/debug-buildfarm-builder.rst

```diff
@@ -26,6 +32,35 @@ To anoint yourself to be part of that team in qastaging, see
 
 3. Click the "Switch to manual-mode" button.
 
+.. note::
+
+   Setting the builder to ``manual`` prevents new builds to be dispatched to
+   the builder, but the container will be removed.
+
+(Alternative) Set a builder to disabled via CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to have the ubuntu-archive-tools installed.
+
+.. code-block:: sh
+
+    $ git clone https://git.launchpad.net/ubuntu-archive-tools
+    $ sudo apt install python3-launchpadlib python3-ubuntutools
+
+
+For example, you want to debug the ``lcy02-amd64-004`` instance.
```

Comment:
I'd add "on qastaging", and I'd mention that this command 'disables' the builder, it's not just a 'debug' command

Reply:
I added `on qastaging` to the headline of the page - I think that is the proper place. Thanks for the hint.